domains/authorization/Authorization.API/API/Authorization/[Features]/Internal/GetConsentForUserCommand.cs

```diff
@@ -73,7 +72,7 @@ public async Task<GetConsentForUserCommandResult> Handle(GetConsentForUserComman
                 subType,
                 command.OrgName,
                 organization.Id,
```

Comment:
OrganizationId er stadigt autogeneret af ef core, upon insert, ikke sandt?

Reply:
_OrganizationId_ is created by _Create_ method in _Organization_ entity.